src/main/java/ch/uzh/ifi/hase/soprafs21/security/SecurityConfig.java

```diff
@@ -31,7 +31,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .and().exceptionHandling()
                 .authenticationEntryPoint((req, rsp, e) -> rsp.sendError(HttpServletResponse.SC_UNAUTHORIZED))
                 .and()
-                .addFilterBefore(new JwtTokenAuthenticationFilter(jwtTokenUtil, userService), UsernamePasswordAuthenticationFilter.class)
+//                .addFilterBefore(new JwtTokenAuthenticationFilter(jwtTokenUtil, userService), UsernamePasswordAuthenticationFilter.class)
```

Comment:
do we need to commit a filter check?

Reply:
I have to comment it when I test http calls.

Reply:
But that can be done on localfor testing